qubes/devices.py

```diff
@@ -433,7 +431,7 @@ class PersistentCollection:
     '''
 
     def __init__(self):
-        self._dict = {}
+        self._dict = OrderedDict()
```

Comment:
[Regular dicts are already ordered these days:](https://docs.python.org/3/library/stdtypes.html#dict)  

"Changed in version 3.7: Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6."

Reply:
I think it is OK to exclude this change. I've found three reasons to use OrderedDict even in modern Python versions:

1. You need some extra method provided by OrderedDict that is missing in dict. It doesn't apply to this PR.
2. Equality (===). It doesn't apply to this PR.
3. Documentation. By using OrderedDict, you make it explicit that you care about the order. I neither strongly support nor strongly oppose this.

Reply:
I dropped OrderedDict then. And also squashed the two commits, so it's easier to follow the history (no more splitting+merging again the function)